### ISSUE TITLE
'Sirenix' could not be found

### ISSUE THREAD

**Issue by missphyrgames** [IvanMurzak/Unity-Gyroscope-Parallax#2]
Hey, I'm getting the following error when I try to edit the manifest.json file.

```
Library\PackageCache\extensions.unity.gyroscope.parallax@1.4.0\Scripts\Mover2D\GyroMover2D.cs(2,7): error CS0246: The type or namespace name 'Sirenix' could not be found (are you missing a using directive or an assembly reference?)
```
Unity version: 2021.3.11f1

**Comment by IvanMurzak** [comment]
@missphyrgames hey. That is fine for cloning the project. I will add instructions on how to clone the project for any reason.
If you want just use it as a user, you don't need to clone it.

**Comment by IvanMurzak** [comment]
Hey, thanks for the report. Fixed in Version 1.4.1. https://github.com/IvanMurzak/Unity-Gyroscope-Parallax/commit/470e05c81f2d9301a82ff54358802d22b70c63af

Also, you may try to install ODIN Inspector plugin, the Unity-Gyroscope-Parallax can use ODIN automatically to make the interface in the Inspector better. https://odininspector.com/
